rest/permissions.py

```diff
@@ -0,0 +1,46 @@
+"""
+Permission classes for REST views
+"""
+
+from __future__ import unicode_literals
+
+from rest_framework.permissions import (
+    BasePermission,
+    SAFE_METHODS,
+)
+from django.http import Http404
+
+from learningresources.models import Repository
+from learningresources.api import (
+    get_repo,
+    PermissionDenied,
+    NotFound,
+)
+
+
+# pylint: disable=protected-access
+class AddRepoPermission(BasePermission):
```

Comment:
As discussed in chat, this name is misleading. When called via `GET` it returns a list of repositories.

If it actually is checking "add repository" permissions, then the "return True" for any user on line 27 is bad (just because they're authenticated doesn't mean they could ever successfully submit an "add repo" request.